In [297]:
from glob import glob
import os

import pandas as pd

In [298]:
# data_dir = f"/media/z/0/MVPC10/DATA/v1.1/RAW/03"
number = "007"
data_dir = f"/home/z/{number}"
path = glob(f"{data_dir}/*/*.png")
path

['/home/z/007/20220826/1661470636493.png',
 '/home/z/007/20220826/1661500276762.png',
 '/home/z/007/20220826/1661473516010.png',
 '/home/z/007/20220826/1661519537045.png',
 '/home/z/007/20220826/1661466256834.png',
 '/home/z/007/20220826/1661493436643.png',
 '/home/z/007/20220826/1661506276402.png',
 '/home/z/007/20220826/1661511556915.png',
 '/home/z/007/20220826/1661512097083.png',
 '/home/z/007/20220826/1661467336890.png',
 '/home/z/007/20220826/1661466136927.png',
 '/home/z/007/20220826/1661475136831.png',
 '/home/z/007/20220826/1661463316895.png',
 '/home/z/007/20220826/1661493856432.png',
 '/home/z/007/20220826/1661498776476.png',
 '/home/z/007/20220826/1661478376473.png',
 '/home/z/007/20220826/1661502856184.png',
 '/home/z/007/20220826/1661516836306.png',
 '/home/z/007/20220826/1661504236550.png',
 '/home/z/007/20220826/1661463976424.png',
 '/home/z/007/20220826/1661476996407.png',
 '/home/z/007/20220826/1661517796188.png',
 '/home/z/007/20220826/1661520197142.png',
 '/home/z/0

In [299]:
len(path)

12513

In [300]:
name = []
for i in path:
    name.append(int(os.path.basename(i).replace(".png", "")))

name

[1661470636493,
 1661500276762,
 1661473516010,
 1661519537045,
 1661466256834,
 1661493436643,
 1661506276402,
 1661511556915,
 1661512097083,
 1661467336890,
 1661466136927,
 1661475136831,
 1661463316895,
 1661493856432,
 1661498776476,
 1661478376473,
 1661502856184,
 1661516836306,
 1661504236550,
 1661463976424,
 1661476996407,
 1661517796188,
 1661520197142,
 1661518756247,
 1661483902741,
 1661473936090,
 1661496976346,
 1661479576107,
 1661500457029,
 1661489176672,
 1661522356569,
 1661483716666,
 1661519956394,
 1661472556179,
 1661478076417,
 1661498956376,
 1661505916617,
 1661506936677,
 1661487137112,
 1661499976329,
 1661472136384,
 1661514737187,
 1661488936960,
 1661493016626,
 1661475736866,
 1661489961616,
 1661502196271,
 1661505256904,
 1661479636436,
 1661499076385,
 1661466796518,
 1661462416596,
 1661474116710,
 1661491097084,
 1661482696472,
 1661478556759,
 1661520317114,
 1661514197107,
 1661493376186,
 1661468116232,
 1661473336722,
 1661508376579,
 1661496

In [301]:
df = pd.DataFrame(name, columns=['path'], dtype=int)

In [302]:
df.columns

Index(['path'], dtype='object')

In [303]:
df.set_index('path', inplace=True)
df.sort_index(inplace=True)

In [304]:
# df.sort_values(by=['path'], inplace=True, ascending=True)

In [305]:
df['label'] = None

In [306]:
df

label
path               
1660942817694  None
1660942876110  None
1660942936465  None
1660942996860  None
1660943056261  None
...             ...
1661954116964  None
1661954176336  None
1661954236769  None
1661954296445  None
1661954356730  None

[12513 rows x 1 columns]

In [307]:
# df.loc[df['path'] == 1652734817313]

In [308]:
df.to_csv(f"{number}.csv")
# df.to_csv(f"MVPC.csv", index=False)

In [309]:
df.dtypes

label    object
dtype: object

001
1652734817313 - 28411
1654005556277 - 44040

002
1655387956476 - 59627

003
1656683956439 - 75210

004
1658066356371 - 90747

005
1659535156882 - 106287

006
1660917556430 - 121867

007
1661954356730 - 134380

$ echo "data, label" > 001.csv

$ find ./001/ -regextype posix-extended -regex '^.*[0-9]{12}.png' | sed 's/.png/,[]/g' | sed 's/^.\/[[:digit:]]\{3\}\/[[:digit:]]\{8\}\///g' >> 001.csv